.travis.yml

```diff
@@ -1,23 +1,17 @@
----
 sudo: false
 dist: trusty
 language: generic
 env:
-  - PDK=release
-  - PDK=nightly
+- PDK=release
+- PDK=nightly
 before_install:
-  - '.travis/install_pdk.sh'
+- ".travis/install_pdk.sh"
 script:
-  - '.travis/test_script.sh'
+- ".travis/test_script.sh"
 branches:
   only:
-    - master
+  - master
```

Comment:
I used the `travis` gem to update the config and this is the yaml format it enforces. 🤷‍♂️